packages/jsii-rosetta/lib/fixtures.ts

```diff
@@ -62,7 +62,7 @@ function loadLiterateSource(
       if (fs.existsSync(jsFile)) {
         return fs.readFileSync(jsFile, { encoding: 'utf-8' });
       }
-      return `Missing literate source file ${literateFileName}`;
+      return `throw new Error("Could not render example: missing literate source file ${literateFileName}");`;
```

Comment:
This is kind of funky - why have `throw new Error`? I think `Could not render example...` is clearer no?